builder/base/kss_builder_base.js

```diff
@@ -997,7 +1028,7 @@ class KssBuilderBase {
    * @param {string|null} pageReference The reference of the current page's root
    *   section, or null if the current page is the homepage.
    * @param {Array} sections An array of KssSection objects.
-   * @param {Object} [context] Additional context to give to the template when
+   * @param {Object} context Additional context to give to the template when
```

Comment:
It looks like context is still optional. Checks if it's undefined before taking action on it.